### PR TITLE
#1209 fix for emulation error in tests run on silicon

### PR DESCRIPTION
**Pull request by joshfischer1108**
This addresses the container emulation issue referenced in #1209 

### For all changes:
- [x] Is there a issue associated with this PR? Is it referenced in the commit message?

- [x] Does your PR title start with `#XXXX` where `XXXX` is the issue number you are trying to resolve? 

- [x] Has your PR been rebased against the latest commit within the target branch (typically main)?

- [x] Is your initial contribution a single, squashed commit?

- [x] Is the code properly formatted with `mvn git-code-format:format-code -Dgcf.globPattern=**/*`?

### For code changes:

- [x] Have you ensured that the full suite of tests is executed via `mvn clean verify`?
- [x] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file?
- [ ] If applicable, have you updated the NOTICE file, including the main NOTICE file?

### Note:
Please ensure that once the PR is submitted, you check GitHub Actions for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: core/src/test/java/org/apache/stormcrawler/protocol/selenium/ProtocolTest.java [hunk 1]
```diff
@@ -51,7 +51,8 @@ public class ProtocolTest extends AbstractProtocolTest {
     private static final Logger LOG = LoggerFactory.getLogger(ProtocolTest.class);
 
     private static final DockerImageName SELENIUM_IMAGE =
-            DockerImageName.parse("selenium/standalone-chrome:120.0");
+            DockerImageName.parse("seleniarm/standalone-chromium:latest")
```
Comment (rzo1) [inline core/src/test/java/org/apache/stormcrawler/protocol/selenium/ProtocolTest.java:54]: Wonder if we can use a fixed tag? Reasoning would be, that "latest" can vary between environments / runs making reproducability difficult.

**Review by rzo1 — APPROVED** [review]

File: core/src/test/java/org/apache/stormcrawler/protocol/selenium/ProtocolTest.java (hunk 1 above)
Comment (rzo1) [reply core/src/test/java/org/apache/stormcrawler/protocol/selenium/ProtocolTest.java:54]: `latest` is `[124.0](https://hub.docker.com/layers/seleniarm/standalone-chromium/124.0/images/sha256-cd64cfbd8aef032ba2695425e7b6077c7b18995cdd9504c4f70c070ea7fba76f?context=explore) atm

Comment (joshfischer1108) [reply core/src/test/java/org/apache/stormcrawler/protocol/selenium/ProtocolTest.java]: Thanks for catching, @rzo1 .  I meant to be explicit on a version number but forgot to switch it before committing 🤦 .  It should be addressed now

**Comment by rzo1** [comment]
Thx @joshfischer1108

**Pull request merged**